src/wp-admin/includes/ajax-actions.php

```diff
@@ -190,9 +190,9 @@ function wp_ajax_wp_compression_test() {
 	if ( ini_get( 'zlib.output_compression' ) || 'ob_gzhandler' === ini_get( 'output_handler' ) ) {
 		// Use `update_option()` on single site to mark the option for autoloading.
 		if ( is_multisite() ) {
-			update_site_option( 'can_compress_scripts', 0 );
+			update_site_option( 'can_compress_scripts', 0, 'off' );
```

Comment:
`update_site_option()` does not support a third parameter for autoload values, which is why it didn't exist here and below.

Reply:
yep I just caught that error